mzLib/Test/TestPeptideWithSetMods.cs

```diff
@@ -930,40 +930,6 @@ public static void CountTargetsWithMatchingDecoys()
             }
         }
 
-        [Test] 
-        public static void TestPeptideWithSetModsReturnsBiomarkersInTopDown()
```

Comment:
Are these tests covered above, or are they not useful for the intended purpose? Are there other top-down cleavage tests that cover them?

Reply:
This is now deprecated. Biomarkers are added during database read and not during digestion.